### ISSUE TITLE
Error creating a task with due_date

### ISSUE THREAD

**Issue by genem2** [black-roland/homeassistant-microsoft-todo#62]
Is anyone seeing an error trying to create a task using the due_date option?

```
service: microsoft_todo.new_task
data: {
"subject": "test",
"list_name": "Tasks",
"due_date": "2021-07-22"
}

```

```
2021-07-22 07:14:24 ERROR (MainThread) [homeassistant.helpers.script.websocket_api_script] websocket_api script: Error executing script. Unexpected error for call_service at pos 1: 'zoneinfo.ZoneInfo' object has no attribute 'zone'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 371, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 571, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1491, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1530, in _execute_service
    await self._hass.async_add_executor_job(handler.job.target, service_call)
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/microsoft_todo/calendar.py", line 140, in handle_new_task
    tasks_api.create_task(subject, list_id, note, due_date, reminder_date_time)
  File "/config/custom_components/microsoft_todo/outlook_tasks_api.py", line 34, in create_task
    "timeZone": self.timezone.zone
AttributeError: 'zoneinfo.ZoneInfo' object has no attribute 'zone'
```

Microsoft To Do,  version: 1.0.3

Home Assistant version | core-2021.7.3
-- | --
installation_type | Home Assistant Container
dev | false
hassio | false
docker | true
virtualenv | false
python_version | 3.9.5
os_name | Linux
os_version | 5.10.28-Unraid
arch | x86_64
timezone | America/Los_Angeles

**Comment by black-roland** [comment]
Thank you for reporting. The issue was fixed in [v1.0.4](https://github.com/black-roland/homeassistant-microsoft-todo/releases/tag/v1.0.4). Please let me know if it works for you.

**Comment by genem2** [comment]
Back in business, thank you.
Even happier to see this component wasn't orphaned!
